### DISCUSSION TITLE
GitHub icon invisible on some backgrounds

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#116831] [New to GitHub]
Can you believe the GitHub icon is invisible on some browser backgrounds / themes?

![Screenshot 2024-04-05 06 11 59](https://github.com/community/community/assets/1230959/82f9d0fe-fc31-416f-ab14-663e768acca6)

<HR>

And if we add it to our browser new tab shortcuts,

![Screenshot 2024-04-05 06 12 58](https://github.com/community/community/assets/1230959/ce55cbe1-4890-4e65-842c-9c38274a62d2)

<HR>

Examining browser history, it's the only website with the invisible icon:

![20240405T184115](https://github.com/community/community/assets/1230959/797aeb5f-baea-4a91-8e80-397325bbbb8e)

<HR>

Update: Some GitHub sites are looking better now,

![Screenshot 2024-04-06 04 43 29](https://github.com/community/community/assets/1230959/c62809c4-81c4-4be1-b468-81c0e5fecd30)

<HR>

We observe some progress compared to above. But they're still not as black as X/Twitter, more of a weak gray,

![20240406T142321](https://github.com/community/community/assets/1230959/7bbeded9-7653-4482-9792-b6b8f5247575)

<HR>

And the tabs are not fully ready yet too,

![20240406T142455](https://github.com/community/community/assets/1230959/1bf49557-5ebc-4d5b-b58f-665ebca8ebdc)

**Comment by Sohailkhan2k02** [comment]
It seems like you're experiencing issues with the visibility or appearance of the GitHub icon on certain browser backgrounds or themes. This can be frustrating, especially when adding GitHub to browser new tab shortcuts.

From your screenshots, it's evident that the GitHub icon appears invisible or poorly contrasted against certain backgrounds. While some progress has been made in terms of visibility, it still falls short compared to other websites like X/Twitter, which have clearer icons and tab appearances.

Here are a few possible explanations and suggestions:

Icon Contrast: The GitHub icon may not have enough contrast against certain background colors or themes, making it difficult to see. This could be due to the design of the icon itself or the theme settings of the browser.

Browser Compatibility: Different browsers may handle icon rendering differently, leading to inconsistencies in visibility. It's possible that the browser you're using has specific rendering issues with the GitHub icon.

GitHub Updates: GitHub periodically updates its interface and design elements, including icons. It's possible that recent updates have affected the visibility or appearance of the GitHub icon on certain backgrounds.

Customization Options: Check if your browser or operating system allows for customization of icon appearances. You may be able to adjust settings to improve the visibility of the GitHub icon.

Feedback to GitHub: Consider providing feedback to GitHub about the visibility issue. They may not be aware of the problem, and user feedback can help prioritize fixes or improvements.

In the meantime, you can try adjusting your browser or system settings to see if it improves the visibility of the GitHub icon. Additionally, keeping an eye on GitHub updates and browser releases may reveal improvements or fixes to the issue over time.

**Comment by jidanni** [comment]
Thanks. Yes, I told GitHub. I just thought they would like to know.
P.S., https://github.com/bitwarden/brand/issues/17
https://twitter.com/TheFrosty/status/1224383002555273216

**Comment by Sohailkhan2k02** [comment]
if you like my answer plz mark as answered

**Comment by jidanni** [comment]
Let's see what GitHub Inc. has to say.

**Comment by jidanni** [comment]
OK, they said
> Thanks for reaching out to GitHub Support. My sincerest apologies for the delay in our response! We appreciate your patience.
> 
> Our favicon loader makes use of the prefers-color-scheme media query to determine the most appropriate colour of our icon. Here's a great article about this particular media query and how it works: 
>  
> https://web.dev/prefers-color-scheme/#the-prefers-color-scheme-media-query
> 
> This query is based on your system's light/dark mode preference, which appears to be set to dark mode in your screenshot. Hence why the dark mode icon is being loaded.
> 
> I'm not sure there's an easy fix for this or a way the site could determine the colour of the browser window it's being loaded in. As you have already posted to the official GitHub community discussions repository you may wish to follow-up there to see if anybody is able to offer any suggestions.
> 
> Please note that while the Product team reads and evaluates all feedback, we cannot guarantee a response to every submission.
> I hope this helps, feel free to get in touch if you have any questions.
> Thanks,
> 
> Steve
> GitHub Support

OK, but all I know is X/Twitter gets it right.
